core/src/test/scala/unit/kafka/security/auth/ZkAuthorizationTest.scala

```diff
@@ -250,6 +249,8 @@ class ZkAuthorizationTest extends ZooKeeperTestHarness with Logging {
     // Check consumers path.
     val consumersAcl = firstZk.getAcl(ConsumerPathZNode.path)
     assertTrue(ConsumerPathZNode.path, isAclCorrect(consumersAcl, false, false))
+    assertTrue("/kafka-acl-extended", isAclCorrect(firstZk.getAcl("/kafka-acl-extended"), secondZk.secure,
+      ZkUtils.sensitivePath(ZkUtils.KafkaAclExtendedPath)))
```

Comment:
ZkUtils is deprecated. We can use ZkData.sensitivePath(path) and ExtendedAclZNode.path.